pkgs/applications/video/obs-studio/default.nix

```
@@ -44,8 +45,9 @@ in mkDerivation rec {
   src = fetchFromGitHub {
     owner = "obsproject";
     repo = "obs-studio";
-    rev = version;
+    rev = "refs/tags/${version}";
     sha256 = "1d502f80whh686mvq0yn6zpa5nvmnlzxwp5sjz43vpbbvhpbrdqj";
```

```suggestion
    sha256 = "vMrPMuZPeFefpbtQykeWifLBkhkSM47tDPid5cU3xa0=";
```
fetching submodules changed the hash and this didn't get attention because we both had the old version in cache probably.